src/__tests__/utils/transform.tsx

```diff
@@ -79,3 +148,22 @@ export const createTransform = (programTransformer: ProgramTransformer) => (
   const actual = ts.transform(sourceFile, [transformer]).transformed[0];
   return printer.printFile(actual).toString();
 };
+
+function createMockModule(name: string, fs: typeof Fs) {
+  mkdirp({ path: `/node_modules/${name}`, fs });
+
+  fs.writeFileSync(`/node_modules/${name}/index.js`, `module.exports = {};`);
+  fs.writeFileSync(
+    `/node_modules/${name}/index.d.ts`,
```

Comment:
is there any way around hardcoding this type def?

Reply:
Sure, did this in my initial push towards an external helper lib: https://github.com/marionebl/ts-transformer-testing-library/blob/master/src/transform-file.ts#L82

If you think this important we can backport it.

Reply:
ah cool - nah not for now - let's just use your lib after it's ready :)